Code/Framework/AzFramework/Platform/Common/Xcb/AzFramework/XcbNativeWindow.cpp

```diff
@@ -6,13 +6,18 @@
  *
  */
 
+#include <AzCore/Settings/SettingsRegistry.h>
+#include <AzCore/std/containers/vector.h>
 #include <AzFramework/Application/Application.h>
+#include <AzFramework/Asset/AssetSystemBus.h>
```

Comment:
Unused include. I think it is not needed?

Reply:
Good catch